main/tests/server/src/com/google/refine/importers/JsonImporterTests.java

```diff
@@ -296,6 +296,52 @@ public void EnsureJSONParserHandlesgetLocalNameCorrectly() throws Exception{
         }
     }
 
+    @Test
+    public void testCanParseTab() throws Exception {
+        // Use un-escaped tab here. 
+        String sampleJson = "{\"field\":\"\tvalue\"}";
+        String sampleJson2 = "{\"\tfield\":{}}";
+        
+        
+        JSONTreeReader parser = new JSONTreeReader(new ByteArrayInputStream(sampleJson.getBytes("UTF-8")));
+        Token token = Token.Ignorable;
+        int i = 0;
+        try {
+            while (token != null) {
+                token = parser.next();
+                if (token == null) {
+                    break;
+                }
+                i++;
+                if (i == 3) {
+                    Assert.assertEquals(Token.Value, token);
+                    Assert.assertEquals("field", parser.getFieldName());
```

Comment:
I don't see anything that tests for the value `"\tvalue"` which what the original bug was about.